src/xref2/component.ml

```diff
@@ -474,11 +486,14 @@ end =
   CComment
 
 and Label : sig
+  (** In order to generate content for links without content *)
+  type content = Heading of Odoc_model.Comment.paragraph | NestableBlock
+
   type t = {
```

Comment:
The attrs have been moved onto the `` `Heading `` constructor right? Any idea why they weren't there to begin with? With your fix it's much more similar to `Comment.block_element` now, so I'm confused why we didn't do that from day one!